### DISCUSSION TITLE
Codespace Port Forwarding does not pass port in the Host header

### DISCUSSION THREAD

**Discussion by AndriiTsok** [community/community#59797] [Codespaces]
### Select Topic Area

Bug

### Body

Host is missing the port value.

<img width="1130" alt="Screenshot 2023-06-30 at 9 27 40 PM" src="https://github.com/community/community/assets/733224/b9b15fac-ed33-41d1-864b-f2f5986ff995">

The other version of port forwarding that @joshspicer mentioned in https://github.com/orgs/community/discussions/57596 works provides the port value in the headers.

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for reporting, good to know that the issue is fixed with newer version of port forwarding, and we do plan to GA it pretty soon.

In the meanwhile, I've opened an internal issue for tracking this issue. I'd respond to this discussion when I have more details.
Thank you!
